dll/ntdll/def/ntdll.spec

```diff
@@ -264,38 +264,38 @@
 @ stdcall NtContinue(ptr long)
 @ stdcall NtCreateDebugObject(ptr long ptr long)
 @ stdcall NtCreateDirectoryObject(long long long)
-@ stub -version=0x600+ NtCreateEnlistment
+@ stdcall -stub -version=0x600+ NtCreateEnlistment(ptr long ptr ptr ptr long long ptr)
 @ stdcall NtCreateEvent(long long long long long)
 @ stdcall NtCreateEventPair(ptr long ptr)
 @ stdcall NtCreateFile(ptr long ptr ptr long long long ptr long long ptr)
 @ stdcall NtCreateIoCompletion(ptr long ptr long)
 @ stdcall NtCreateJobObject(ptr long ptr)
 @ stdcall NtCreateJobSet(long ptr long)
 @ stdcall NtCreateKey(ptr long ptr long ptr long long)
-@ stub -version=0x600+ NtCreateKeyTransacted
+@ stdcall -stub -version=0x600+ NtCreateKeyTransacted(ptr long ptr long ptr long long ptr)
 @ stdcall NtCreateKeyedEvent(ptr long ptr long)
 @ stdcall NtCreateMailslotFile(long long long long long long long long)
 @ stdcall NtCreateMutant(ptr long ptr long)
 @ stdcall NtCreateNamedPipeFile(ptr long ptr ptr long long long long long long long long long ptr)
 @ stdcall NtCreatePagingFile(ptr ptr ptr long)
 @ stdcall NtCreatePort(ptr ptr long long ptr)
-@ stub -version=0x600+ NtCreatePrivateNamespace
+@ stdcall -stub -version=0x600+ NtCreatePrivateNamespace(ptr long ptr ptr)
 @ stdcall NtCreateProcess(ptr long ptr ptr long ptr ptr ptr)
 @ stdcall NtCreateProcessEx(ptr long ptr ptr long ptr ptr ptr long)
 @ stdcall NtCreateProfile(ptr ptr ptr long long ptr long long long) ; CHECKME
-@ stub -version=0x600+ NtCreateResourceManager
+@ stdcall -stub -version=0x600+ NtCreateResourceManager(ptr long ptr ptr ptr long wstr)
 @ stdcall NtCreateSection(ptr long ptr ptr long long ptr)
 @ stdcall NtCreateSemaphore(ptr long ptr long long)
 @ stdcall NtCreateSymbolicLinkObject(ptr long ptr ptr)
 @ stdcall NtCreateThread(ptr long ptr ptr ptr ptr ptr long)
-@ stub -version=0x600+ NtCreateThreadEx
+@ stdcall -stub -version=0x600+ NtCreateThreadEx(ptr long ptr ptr ptr ptr long long long long long)
```

Comment:
I am not sure if it's a good idea to have this as 'succeeding' stub?
That might cause quite hard to track down issues

Reply:
Good catch, fixed.